Campaign.html

```diff
@@ -1,188 +0,0 @@
-var myfont_webfont_test = false;
-// change this to false if you're having trouble with WOFFs
-var woffEnabled = true;
-// to place your webfonts in a custom directory, uncomment this and set it to where your webfonts are.
-//var customPath = "/themes/fonts";
-</script>
-<script type="text/javascript" src="MyFontsWebfontsOrderM2858474.js"></script>
-<link href="style.css" rel="stylesheet" type="text/css" />
-
-    <script type="text/javascript">
-
-  var _gaq = _gaq || [];
-  _gaq.push(['_setAccount', 'UA-21871866-1']);
-  _gaq.push(['_trackPageview']);
-
-  (function() {
-    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-  })();
-
-</script>
-  
-</head>
-   
-         
-         
-  
-<body>
-<div class="wrapper">
-<div id="jar"><br />
-	<div id="mast">
-    	<div class="District-Book" id="left"><a href="visual.html" title="Visual Design ">Visual</a> <a href="web.html" title="Web and Interactive Design">web/interactive</a>  <a href="photo.html" title="Digital Photography">photo</a></div>
-    <div id="logo"><a href="index.html"><img src="spacer.png" width="327" height="61" class="spacer" /></a></div>
-   	 <div class="District-Book" id="right"><a href="motion.html" title="Motion Graphics/ Digital Video">Motion</a> <a href="Campaign.html" title="Campaign Design">Campaign</a> <a href="resume.pdf" title="My Resume">My<span class="space">/</span>ResumÉ</a> </div>
-   		 </div>
-    <br />
-    <br />
-    <div id="main">  <hr class="hr3" />
-
-    
-        
-    <div class="desc"><h2>Craigslist Rebranding Campaign</h2>
-    <p>For this academic assignment the objective was to create a friendly brand concept for the popular online community/ classifieds website Craigslist.com. Creating a more user friendly web site, and a unified campaign helped polish the companies current reputation while still upholding their "non-corporate" mission.<br /> <em>HTML, CSS, Javascript, After Effects, Logic Pro, Soundtrack Pro, Photoshop</em></p></div>
-    <img name="" src="cllogo.jpg"/> 
-   <br />
-
-    <div class="rollover2_img"><a title="Craigslist" href="http://www.vannessaphillips.com/craigslist/index.html" target="_new">
-<span>Launch Website </span>
-</a></div>
-<div align="right"><span class="launch"><a href="http://www.vannessaphillips.com/craigslist/index.html" target="_new">Launch+</a></span></div>
-   <br />
-<img src="clads.jpg" class="border"><br />
-<img src="EN1.jpg"><br />
-<img src="en2.jpg" class="border"><br />
-<img src="stationerycl.jpg" alt="" name="stationery" width="900" height="675"><br />
-
-<iframe src="http://player.vimeo.com/video/23100178?title=0&amp;byline=0&amp;portrait=0&amp;color=f29183" width="900" height="675" frameborder="0"></iframe>
-<br />
```

Comment:
in general, I recommend using CSS to add spacing, not multiple brs - br should really only be used when you want to break a line of text into parts